src/EditorFeatures/CSharpTest/ChangeSignature/ChangeSignatureTests.cs

```diff
@@ -335,7 +335,29 @@ public class C3
                     }
                     else if (updatedDocument.Name == "C3.cs")
                     {
-                        // shouldn't change unchangeable document
+                        // change should be in new solution in unchangeable document
+                        Assert.Contains("bool _x = C1.M(2, 1);", (await updatedDocument.GetTextAsync(CancellationToken.None)).ToString());
+                    }
+                }
+
+                // Actually apply the change to workspace
+                Assert.True(testState.Workspace.TryApplyChanges(result.UpdatedSolution));
+
+                foreach (var updatedDocument in testState.Workspace.CurrentSolution.Projects.SelectMany(p => p.Documents))
+                {
+                    if (updatedDocument.Name == "C1.cs")
```

Comment:
See comment also for other tests: let's change this to just fetch each document by name and then assert with it. The risk of this is the loop could potentially be skipping all the Assert.Contains() if the document names aren't what you expect them to be. The earlier code probably should be changed too. :-)

Reply:
this test is removed